setup.py

```diff
@@ -5,7 +5,7 @@
 
 
 install_requires = (
-    'djangorestframework>=3.1.0,<3.2',
+    'djangorestframework>=3.2.0,<3.3',
```

Comment:
Only if you've broken compatibility. If `3.1` still works, just change the `>=3.2.0,<3.3` to `>=3.1.0,<3.3`.

We could also include the DRF version in the tox setup for travis. That would double the test runs though...